code/modules/client/preference/preferences_toggles.dm

```diff
@@ -36,6 +36,16 @@
 	to_chat(usr, "You will [(prefs.toggles & CHAT_RADIO) ? "now" : "no longer"] see radio chatter from radios or speakers")
 	feedback_add_details("admin_verb","THR") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/verb/toggle_ai_voice_annoucements()
+	set name = "Hear/Silence AI Voice Announcements"
+	set category = "Preferences"
+	set desc = "Toggle hearing AI annoucements in voice form or in text form"
+	if(!holder)	return
+	prefs.sound ^= SOUND_AI_VOICE
+	prefs.save_preferences(src)
+	to_chat(usr, "[(prefs.sound & SOUND_AI_VOICE) ? "You will now hear AI announcements." : "AI annoucements will now be converted to text."] ")
+	feedback_add_details("admin_verb","TAIvoice") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```

Comment:
feedback_add_details isn't needed because it's for the old blackbox system

Reply:
@Ty-Omaha The new blackbox system isnt in yet, so it is still required

Reply:
@AffectedArc07 The blackbox isnt even used, we don't need a new one. Look at all the new preference toggles without it.

Reply:
It does look like *some* of the preference verbs aren't using the `feedback_add_details` line, so I suppose it could work without it. But I'm not familiar with this system, and if it matters that I remove it.